webapp/channels/src/components/pricing_modal/content.tsx

```diff
@@ -234,7 +249,7 @@ function Content(props: ContentProps) {
     };
 
     const enterpriseBtnDetails = () => {
-        if (isPostTrial && isAdmin) {
+        if (enabled || (isPostTrial && isAdmin)) {
```

Comment:
What will it show after we enable cloud free deprecation?

Reply:
It shows contact sales cta